integration_test/workflow_tests/test_packages/ingest/test_multiple_granules_happy_path.py

```diff
@@ -161,10 +172,18 @@ def test_multiple_granules_happy_path(self):
                         head_object_output["ResponseMetadata"]["HTTPStatusCode"],
                         f"Error searching for object {key} in the {recovery_bucket_name}",
                     )
+                    self.assertEqual(
+                        "GLACIER",
+                        head_object_output["StorageClass"]
+                    )
                     s3_versions.append(head_object_output["VersionId"])
             except Exception as ex:
-                return ex
+                logging.error(ex)
+                raise
 
+            # Let the catalog update
+            time.sleep(1)
```

Comment:
is this necessary?

Reply:
I was getting random errors when I left it out. I think it was due to post_to_catalog not completing yet.